pom.xml

```diff
@@ -53,6 +53,8 @@
 
     <properties>
         <dropwizard.url>http://www.dropwizard.io/${project.version}</dropwizard.url>
+        <dropwizard.check.fail-owasp>true</dropwizard.check.fail-owasp>
```

Comment:
No need to introduce another property here. The OWASP dependency check plugin provides the `failOnError` and `failBuildOnAnyVulnerability` properties out of the box.